### ISSUE TITLE
achievement and progress feature

### ISSUE THREAD

**Issue by hardikhere** [fnplus/footsteps-app#157]
**user can not see how many paths he/she has completed and his progress in particular path**
It would be cool is user can see how much he/she has achieved and may win badge of completion of paths he/she has covered.

**solution**
we have to add information about completed paths and progress of current path in user model and have to make a separate page to show that 
:)

**Comment by hardikhere** [comment]
I would love to work on that

**Comment by xlogix** [comment]
Hey there, we can definitely add completed paths in a separate page but maybe we can add the progress bar on the path itself

**Comment by hardikhere** [comment]
![image](https://user-images.githubusercontent.com/51117080/82550737-df399c00-9b13-11ea-9c37-b12779d99644.png)

can you please tell me why I am facing this error i have been trying a lot

**Comment by xlogix** [comment]
Have you followed all the instructions as expected?

**Comment by hardikhere** [comment]
Yes I followed all instruction.. its saying that it is unable to find .cache/match-path.json..
I have also tried to clear my npm cache but not working

**Comment by xlogix** [comment]
Hey @hardikhere, can you try it now?

**Comment by hardikhere** [comment]
> Hey @hardikhere, can you try it now?
Can I try this weekend?

**Comment by xlogix** [comment]
Sure
